src/main/java/in/chamberl/team/walletcheck/crew/Crew.java

```diff
@@ -2,11 +2,18 @@
 
 import org.springframework.shell.standard.ShellMethod;
 import org.springframework.shell.standard.ShellComponent;
+import java.io.UnsupportedEncodingException;
 
 @ShellComponent
 public class Crew {
 	@ShellMethod("Crew's command.")
-	public String crew(String input) {
-		return "input was:" + input;
+	public int crew(String input) throws UnsupportedEncodingException {
+		byte[] crewsstringsize = input.getBytes("US-ASCII");
+		return crewsstringsize.length;
+	}
+@ShellMethod("Crew's command.")
+	public byte[] kyle(String input) throws UnsupportedEncodingException {
+		byte[] crewsstringsize = input.getBytes("US-ASCII");
```

Comment:
your variable name `crewstringsize` doesn't really make sense, since it is a `byte[]` or a list of bytes. `crewstringsize.length` is a size, so I would recommend cleaning this up like this.


```java
byte[] inputAsBytes = input.getBytes("US-ASCII");
int inputLength = inputAsBytes.length;
return inputLength;
```